backend/src/main/java/com/redhat/cloud/notifications/routers/EventResource.java

```diff
@@ -64,6 +66,7 @@ public Page<EventLogEntry> getEvents(@Context SecurityContext securityContext, @
 
         Set<EndpointType> basicTypes = Collections.emptySet();
         Set<CompositeEndpointType> compositeTypes = Collections.emptySet();
+        Set<NotificationStatus> notificationStatusSet = status == null ? Set.of() : toNotificationStatus(status);
```

Comment:
I noticed that `status` was always an empty set, even if the query param wasn't specified- Anyway added a null check.